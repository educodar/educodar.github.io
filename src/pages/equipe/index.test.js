import React from "react"
import renderer from "react-test-renderer"

import TeamPage from "../equipe"

describe("Team page", () => {
  /**
   * @todo fix failing test: TeamPage (...): Nothing was returned from render.
   */
  xit("renders correctly", () => {
    const tree = renderer.create(<TeamPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
