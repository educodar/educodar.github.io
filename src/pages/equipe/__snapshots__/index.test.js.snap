// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Team page renders correctly 1`] = `
<article
  className="MuiBox-root MuiBox-root-2"
>
  <h1
    className="MuiTypography-root MuiTypography-h1"
  >
    Quem faz o Educodar
  </h1>
  <h2
    className="MuiTypography-root MuiTypography-h2"
  >
    Educodar Belo Horizonte
  </h2>
  <ul
    className="MuiList-root MuiList-padding"
  >
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Jimmy Andrade (Belo Horizonte) - Champion de Comunicação
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Leandro Victor (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Thais Nunes (Belo Horizonte) - Champion de Seleção de Turma
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Vagner dos Santos (Belo Horizonte)
    </li>
  </ul>
  <h2
    className="MuiTypography-root MuiTypography-h2"
  >
    Site Educodar.com.br
  </h2>
  <ul
    className="MuiList-root MuiList-padding"
  >
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Daniel Nascimento (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Inajara Leppa (Porto Alegre)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Jimmy Andrade (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Lucas Carvalhais (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Vinícius Ponciano (Belo Horizonte)
    </li>
  </ul>
  <h2
    className="MuiTypography-root MuiTypography-h2"
  >
    Já colaboraram
  </h2>
  <ul
    className="MuiList-root MuiList-padding"
  >
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      André Araújo (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Bruna Castelo (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Luciano Borges (Belo Horizonte) - Suporte e Orçamento
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Maria Simas (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Nayara Souza (Belo Horizonte)
    </li>
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      Paulo Gama (Belo Horizonte e Recife)
    </li>
  </ul>
</article>
`;
